package.json

```diff
@@ -21,7 +21,8 @@
     }
   ],
   "dependencies": {
-    "grunt": "0.4.5",
-    "grunt-jquery-content": "3.0.0"
+    "grunt": "1.6.1",
+    "grunt-cli": "1.4.3",
```

Comment:
grunt-cli should not be needed. Afaik we only invoke `grunt` via `npm run …` or `node_modules/.bin/`, which is already included in grunt.

If we need it globally outside npm, then the dependency wouldn't do that. In most repos, I've ended up removing this over the years. Looks like we forgot to remove it from api.jqueryui.com, and the state of this old branch suggests there was even a time where we didn't yet specify it even.

Long story short: Can we do it without this?

Reply:
I'm not quite sure what I did wrong the first time I tested this, but yes, looks like this works correctly without an explicit `grunt-cli` require. Updated.